docs/guides/noise-learning.ipynb

```diff
@@ -0,0 +1,303 @@
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f'Noise learner result contains {len(noise_model.data)} entries'\\\n",
+    "       f' and has the following type:\\n {type(noise_model)}\\n')\n",
+    "print(f'Each element of `NoiseLearnerResult` then contains'\\\n",
+    "       f' an object of type:\\n {type(noise_model.data[0])}\\n')\n",
+    "print(f'And each of these `LayerError` objects possess the'\\\n",
+    "      f' following data: \\n{noise_model.data[0].error}\\n')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a27a38f1-393d-49f9-92c3-a8634bfb1a60",
+   "metadata": {},
+   "source": [
+    "The `LayerError.error` attribute of the noise learning result contains the generators and error rates of the fitted Pauli Lindblad model, which has the form:\n",
+    "\n",
+    "$$ \\Lambda(\\rho) = \\exp{\\sum_j r_j \\left(P_j \\rho P_j^\\dagger - \\rho\\right)} $$\n",
+    "\n",
+    "where the $r_j$ are the `LayerError.rates` and $P_j$ are the Pauli operators specified in `LayerError.generators`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7ebb4b7d-ef45-4996-bf5a-125d0831ebca",
+   "metadata": {},
+   "source": [
+    "## Noise learning options\n",
+    "\n",
+    "There are a few options available to input into a `NoiseLearner` object, can be set with when instantiated. These are encapsulated by the `qiskit_ibm_runtime.options.NoiseLearnerOptions` class and include the ability to specify the maximum layers to learn, number of randomizations, and the twirling strategy, among others. Refer to the API documentation about [`NoiseLearnerOptions`](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.NoiseLearnerOptions) for more detailed information.\n",
```

Comment:
```suggestion
    "There are a few options available to input into a `NoiseLearner` object, can be set with when instantiated. These are encapsulated by the `qiskit_ibm_runtime.options.NoiseLearnerOptions` class and include the ability to specify the maximum layers to learn, number of randomizations, and the twirling strategy, among others. Refer to the API documentation about [`NoiseLearnerOptions`](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.NoiseLearnerOptions) for more detailed information.\n",
```

Something feels off grammatically with this sentence?

Reply:
Yeah, looks like I accidentally left a few garbled words from a previous draft of that sentence. Should be fixed now.